packages/gestalt/src/Video.js

```diff
@@ -549,6 +551,8 @@ export default class Video extends PureComponent<Props, State> {
             onSeeked={this.handleSeek}
             onTimeUpdate={this.handleTimeUpdate}
             onProgress={this.handleProgress}
+            // $FlowIssue facebook/flow#8448
```

Comment:
facebook/flow#8448

Reply:
@valbooth28 thanks for the PR! Made a few updates:
* Remove the $FlowIssue - see https://github.com/facebook/flow/issues/8186#issuecomment-555747680
* Updated the docs to include the `objectFit` prop
* Added tests to ensure objectFit gets rendered

Also know that with every flow issue, we require the exact exception, so instead of something like `// $FlowFixMe`, we should write `// $FlowFixMe[prop-missing]`. See https://github.com/gajus/eslint-plugin-flowtype/issues/453 for the request to make this an ESLint rule